### ISSUE TITLE
Not able to make it work on windows

### ISSUE THREAD

**Issue by rnikhil275** [sailorproject/sailor#109]
When I visit "localhost/sailor" it just print out what is inside the index.lua file. I added everything mentioned [here](https://github.com/sailorproject/sailor/blob/master/docs/INSTALL_WIN.md) in the apache conf file. 

My conf [file](http://pastebin.com/1K246wL7)
My error [log](http://pastebin.com/51VnJ1bg)

I can't seem to figure out where it's going wrong. I have Lua 5.1 installed.

**Comment by felipedaragon** [comment]
Hello! I noticed from the log file it mentions "Apache/2.4.18 (Win64)". In this case you will need the Lua C libraries compiled for Win64 - the libraries included with the Lua For Windows installer, for example, are 32-bit.

**Comment by felipedaragon** [comment]
You should consider a parallel 32-bit installation of Apache/2.4.18 - just make sure the LuaPackageCPath points to the right path where Lua for Windows is installed and it should work as expected

**Comment by rnikhil275** [comment]
Okay. I downloaded the 32 bit version of Apache. I still get an error. 
[Here ](http://pastebin.com/6Pej7zTD)is my error log

**Comment by felipedaragon** [comment]
Could Lua have been installed to C:/Program Files (x86)/Lua instead of C:/Program Files/Lua/ as indicated in LuaPackage* directives?

**Comment by rnikhil275** [comment]
I was about to edit my previous comment with the updated error log. I noticed my mistake and changed it. This is the new [log](http://pastebin.com/YmgAtiWb)

**Comment by rnikhil275** [comment]
By any chance has it got something to do with backward(\) slashes used in windows file paths instead of forward(/)  ?

**Comment by felipedaragon** [comment]
It looks like you need some missing dependencies in the 'C:/Program Files (x86)/Lua/5.1/lua/' directory, this one: https://github.com/sailorproject/remy/tree/master/src

**Comment by rnikhil275** [comment]
Yes. Noticed that in the logs. I was missing latclient also. Now, after installing both, when I point my browser at the Sailor sample app it just says "webpage not available" and "connection reset". Any pointers on that ?

**Comment by felipedaragon** [comment]
Can you post the new log? These dependencies were included with this repo in the past. Now they are installed through luarocks, this is why they are missing - an updated version of the doc will mention this.

**Comment by rnikhil275** [comment]
[Logs](http://pastebin.com/DQ1y5qT4)

**Comment by felipedaragon** [comment]
Do you have https://github.com/sailorproject/lua_at_client/tree/master/lua?

**Comment by rnikhil275** [comment]
Yes. I installed it using luarocks. [This](https://luarocks.org/modules/etiene/latclient) one right ?

**Comment by felipedaragon** [comment]
Yes, 0.4-1 is the latest. Please feel free to update the docs. Your contribution is very welcome. I'm not sure what is wrong now - the log is not indicating the issue... I think.

**Comment by rnikhil275** [comment]
I think I will just downgrade my Lua installation in my Linux system and poke around Sailor using that

**Comment by felipedaragon** [comment]
Can you post a more clean version of the log without the previous errors that were fixed?

**Comment by rnikhil275** [comment]
This is the final [log](http://pastebin.com/DvVFQTh6). I am not able to find anything relevant from them

**Comment by felipedaragon** [comment]
I will try to reproduce this later today. Are you using Windows 10 64-bit?

**Comment by rnikhil275** [comment]
No. Windows 7. 64 bit

**Comment by rnikhil275** [comment]
I am going to try a clean install of lua/luarocks now. Hopefully that solves the problem

**Comment by rnikhil275** [comment]
I am going to try a clean install of lua/luarocks now. Hopefully that solves the problem

**Comment by ounos** [comment]
@rnikhil275 I didn't check your error logs (too lazy, sorry :P) but I did checked your conf file.

Under `<Directory "c:/Apache24/htdocs">`, you should change `AllowOverride None` to `AllowOverride All`. That is definitely an issue there (probably not the only one). You should restart apache after that.

**Comment by rnikhil275** [comment]
I am sorry I don't have access to a windows machine now. It's at my home and I am staying at college hostel since my exams are going on. I cannot provide any updates on the issue for another 5 days. :worried:
